### PR TITLE
[#12157] fix : misaligned button behaviour

### LINKED ISSUE #12157: Instructor help page: Preview as Student/Instructor buttons misaligned at small screen sizes
**Expected behaviour**
Screenshot (83)

**Actual behaviour**
Screenshot (84)

### PR DESCRIPTION
**Pull request by sumitshinde-84**
Fixes  #12157

**PR Checklist**
i have fix issue and set that button as i shown in issue expected behaiviour image

Ensure that you have:
- [x] Read and understood our PR guideline: https://teammates.github.io/teammates/process.html#step-4-submit-a-pr
  - [x] Added the issue number to the "Fixes" keyword above
  - [x] Titled the PR as specified in the abovementioned document
- [x] Made your changes on a branch other than `master` and `release`
- [x] Gone through all the changes in this PR and ensured that:
  - [x] They addressed one (and only one) issue
  - [x] No unintended changes were made
- [x] Run and passed static analysis: `./gradlew lint` and `npm run lint`
- [] Added/updated tests, if changes in functionality were involved
- [] Added/updated documentation to public APIs (classes, methods, variables), if applicable

**Outline of Solution**

i have added inline style to instructer-preveiw  button

### CODE REVIEW

**Review by zhaojj2209 — CHANGES_REQUESTED** [review]
Please modify your PR title and description to fit our [development guidelines](https://teammates.github.io/teammates/process.html#step-4-submit-a-pr).

Also, what you have modified in this PR is a snapshot file, which is automatically generated and is not meant to be edited manually.

File: #serve# [hunk 1]
```diff
@@ -0,0 +1 @@
+@@ -0,0 +1 @@
```
Comment (zhaojj2209) [inline #serve#:1]: What is this file for? Please remove unnecessary changes.

File: package.json [hunk 2]
```diff
@@ -77,6 +77,7 @@
     "stylelint": "^13.13.1",
     "stylelint-config-standard": "^22.0.0",
     "typescript": "~4.7.4",
-    "webpack-bundle-analyzer": "^4.8.0"
+    "webpack-bundle-analyzer": "^4.8.0",
```
Comment (zhaojj2209) [inline package.json:80]: Fixing this issue does not require additional packages. Please revert changes to `package.json` and `package-lock.json`.

File: src/web/app/components/preview-session-panel/preview-session-panel.component.html [hunk 3]
```diff
@@ -30,7 +30,7 @@
         <ng-container *ngTemplateOutlet="previewInstructorBtn"></ng-container>
       </a>
       <ng-template #previewInstructorBtn>
-        <button id="btn-preview-instructor" class="btn btn-primary" [disabled]="emailOfInstructorToPreview.length === 0 || forDisplayOnly">Preview as Instructor</button>
+        <button style="position:absolute;right:48px" id="btn-preview-instructor" class="btn btn-primary" [disabled]="emailOfInstructorToPreview.length === 0 || forDisplayOnly">Preview as Instructor</button>
```
Comment (zhaojj2209) [inline src/web/app/components/preview-session-panel/preview-session-panel.component.html:33]: Fix not accepted.
image

**Review by zhaojj2209 — CHANGES_REQUESTED** [review]
Please fix all tests before requesting for review.

File: src/web/app/components/preview-session-panel/preview-session-panel.component.html (hunk 3 above)
Comment (sumitshinde-84) [reply src/web/app/components/preview-session-panel/preview-session-panel.component.html:33]: Just tell me how I can run it locally hence I can fix it properly

Comment (zhaojj2209) [reply src/web/app/components/preview-session-panel/preview-session-panel.component.html:33]: Our [developer guidelines](https://teammates.github.io/teammates/setting-up.html) contains all the information you need.
